pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -66,7 +66,7 @@ type PodAutoscalerSpec struct {
 	// So, we add Generation here. Once that gets fixed, remove this and use
 	// ObjectMeta.Generation instead.
 	// +optional
-	Generation int64 `json:"generation,omitempty"`
+	DeprecatedGeneration int64 `json:"generation,omitempty"`
```

Comment:
Could you also update these comments preceding the generation and maybe point to our github issue tracking this work? Here and elsewhere.

Reply:
Sounds good